src/extension/src/generationExperience.ts

```diff
@@ -87,20 +89,19 @@ export class GenerationExperience extends WizardServant {
       payload: { outputPath: enginePayload.path }
     });
 
-    if (payload.selectedFunctions) {
-      serviceQueue.push(
+    if ((payload.selectedFunctions && payload.functions.resourceGroup === "")) {
+      // Reason for this queue is for deploying more than 1 resource group
+      resourceGroupQueue.push(
```

Comment:
Is this gonna be used more so when cosmos is also added?

Reply:
Yes, this is going to be used when cosmos is added